platform/podjs/cypress/integration/pod_spec.js

```diff
@@ -4,3 +4,46 @@ import "../../dist/pod.js";
 describe("pod.js satisfies spec", () => {
     podSpec(window.pod, "");
 });
+
+describe("Import Archive", () => {
```

Comment:
The `pod_spec.js` file is meant for running the spec tests defined within _feature-api_ - so I think this warrants a separate file.